lib/gnulib.mk.in

```diff
@@ -1091,9 +1091,7 @@ endif
 ## begin gnulib module crypto/md5
 ifeq (,$(OMIT_GNULIB_MODULE_crypto/md5))
 
-libgnu_a_SOURCES += md5.c
-
-EXTRA_DIST += gl_openssl.h md5.h
+EXTRA_DIST += md5.h
```

Comment:
This file is autogen'd, @birkenfeld should we be regenerating this via gnulib-tool, are we ok with just raw editing this file for the remacs project?